diff --git a/features/step_definitions/steps.js b/features/step_definitions/steps.js
index 5392532..c4dbf9a 100644
--- a/features/step_definitions/steps.js
+++ b/features/step_definitions/steps.js
@@ -3,12 +3,8 @@ const assert = require('assert')
 
 const isItFriday = today => today === 'Friday' ? 'TGIF' : 'Nope'
 
-Given('today is Sunday', () => {
-	this.today = 'Sunday'
-})
-
-Given('today is Friday', () => {
-	this.today = 'Friday'
+Given('today is {string}', givenDay => {
+  this.today = givenDay;
 })
 
 When('I ask whether it\'s Friday yet', () => {
