diff --git a/features/is_it_friday_yet.feature b/features/is_it_friday_yet.feature
index c96fba1..3a57a3b 100644
--- a/features/is_it_friday_yet.feature
+++ b/features/is_it_friday_yet.feature
@@ -5,3 +5,8 @@ Feature: Is it Friday yet?
 		Given today is Sunday
 		When I ask whether it's Friday yet
 		Then I should be told "Nope"
+
+	Scenario: Friday is Friday
+		Given today is Friday
+		When I ask whether it's Friday yet
+		Then I should be told "TGIF"
