diff --git a/features/is_it_friday_yet.feature b/features/is_it_friday_yet.feature
index 3a57a3b..f1e62f0 100644
--- a/features/is_it_friday_yet.feature
+++ b/features/is_it_friday_yet.feature
@@ -1,12 +1,13 @@
 Feature: Is it Friday yet?
 	Everybody wants to know when it's Friday
 
-	Scenario: Sunday isn't Friday
-		Given today is Sunday
+	Scenario Outline: Today is or is not Friday
+		Given today is "<day>"
 		When I ask whether it's Friday yet
-		Then I should be told "Nope"
+		Then I should be told "<answer>"
 
-	Scenario: Friday is Friday
-		Given today is Friday
-		When I ask whether it's Friday yet
-		Then I should be told "TGIF"
+	Examples:
+		| day            | answer |
+		| Friday         | TGIF   |
+		| Sunday         | Nope   |
+		| anything else! | Nope   |
